wewatch-client/src/App.js

```diff
@@ -20,6 +21,7 @@ const App = () => {
       <Route path="/swiping" component={Swiping}/>
       <Route path="/match" component={Match}/>
       <Route path="/noMatch" component={noMatch}/>
+      <Route path="/error" component={PageErrorIndicator}/>
```

Comment:
do you think it would be easier to just show the user an `alert` when movieList.length==0? because in this case they are taken to a different page and have to restart the lobby. Let me know your thoughts and we can discuss!

Reply:
Hmm right, I thought about this too, however, we would require the filtering process to restart anyways, so I thought it was a good option to direct these users to a different URL. Additionally, this error path would be for the case that we have an empty movieList and the program is looking for an element in that empty list in the swiping container. Let me know your thoughts!